api/server/modules/generic.js

```diff
@@ -0,0 +1,59 @@
+var date = require('date-and-time');
+var bcrypt = require('bcrypt');
+
+var mongo_ObjectID = require('mongodb').ObjectID;
+
+module.exports = {
+    currentTime: function () {
```

Comment:
there is a node library called momenjs use that for all time related operations instead of new Date()